src/router/router.js

```diff
@@ -4,13 +4,15 @@ import { BrowserRouter, Route, Switch } from "react-router-dom";
 import paths from "services/shared/router-paths";
 import Dashboard from "../pages/dashboard";
 import Simulation from "../pages/simulation";
+import Login from "../pages/login";
```

Comment:
Geen imports met ../ 
```suggestion
import Login from "pages/login";
```